src/topology/algebra/order/basic.lean

```diff
@@ -980,8 +980,27 @@ lemma tendsto_nhds_bot_mono' [topological_space β] [partial_order β] [order_bo
 tendsto_nhds_bot_mono hf (eventually_of_forall hg)
 
 section linear_order
+variables [topological_space α] [linear_order α]
 
-variables [topological_space α] [linear_order α] [order_topology α]
+section order_closed_topology
+variables [order_closed_topology α] {a b : α}
+
+lemma eventually_le_nhds (hab : a < b) : ∀ᶠ x in 𝓝 a, x ≤ b :=
+eventually_iff.mpr (mem_nhds_iff.mpr ⟨Iio b, Iio_subset_Iic_self, is_open_Iio, hab⟩)
```

Comment:
You could use `eventually_lt_nhds` to shorten the proof of this one. But it's already a one-liner anyway, so that's not very important.

Reply:
Yeah, this is one of those situations where I feel like copy pasta actually elucidates the key points, which is fine for a one liner.